### ISSUE TITLE
Gitlab support

### ISSUE THREAD

**Issue by sobolevn** [imsky/pull-review#99]
Hi @imsky, thanks for building this awesome bot.

Do you have any plans in supporting Gitlab anytime soon? 
I understand that this feature is rather complex one, but I hope many people will it.

**Comment by imsky** [comment]
@sobolevn yes, i'd definitely want pull-review to support all major Git hosting services. i think GitLab will be the first non-GitHub provider supported.

**Comment by sobolevn** [comment]
@imsky I can provide several hours to help you. I am planning to make this bot an important part of our stack: https://github.com/wemake-services/kira

Please, feel free to assign me on any kind of manual work 🙂

**Comment by imsky** [comment]
ok @sobolevn i appreciate it. i'll definitely reach out when the time comes
